<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>274</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>231</width>
    <height>274</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>231</width>
    <height>274</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">Widget{
color: #000;
background: #fff;
}

QPushButton{
color: #fff;
border-radius: 4px;
background-color: #428bca;
border-color: #357ebd;
}

QPushButton:disabled {
background-color: #326999;
border: 1px solid #2b6699;
color: #aaa;
}

QLineEdit:focus {
border: 1px solid #357ebd;
}
QTextEdit:focus {
border: 1px solid #357ebd;
}</string>
  </property>
  <widget class="QTextEdit" name="keywords">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>211</width>
     <height>70</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="target_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="target_group_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Target group:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sessions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="session_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Current session:</string>
   </property>
  </widget>
  <widget class="QLabel" name="keywords_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Keywords:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_session">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>New session</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
